packages/api-explorer/src/Doc.jsx

```diff
@@ -353,7 +356,6 @@ Doc.propTypes = {
   }),
   auth: PropTypes.shape({}).isRequired,
   baseUrl: PropTypes.string,
-  changeGroup: PropTypes.func.isRequired,
```

Comment:
Renamed this throughout so it's clearer that this is intended for an `onChange` event.